packages/ts-api/.example-env

```diff
@@ -0,0 +1,5 @@
+PORT=8000
```

Comment:
I think this file should be renamed to `.pg-env` or `postgres-env` or something similar.

Reply:
well this file will be renamed by new devs and turned into their actual .env file. I should put that in the readme though!